packages/openapi3/src/openapi.ts

```diff
@@ -380,16 +381,22 @@ function createOAPIEmitter(program: Program, options: OpenAPIEmitterOptions) {
 
           bodyModel = prop.type;
         }
+        if (isStatusCode(program, prop)) {
```

Comment:
I believe the same changes are needed in cadl-autorest. In the future, it would be preferable to make the changes together and follow https://github.com/Azure/cadl-azure/blob/main/CONTRIBUTING.md#making-a-cross-cutting-change-across-both-repos.

Reply:
Yes … sorry. I’m working on a PR to Cadl-autorest now and hope to have that ready very soon.